index.md

```diff
@@ -1,9 +1,68 @@
+This is an extremely useful way to anchor the observatory outputs and science users should be encouraged to cite the instrument DOI explicitly.
+
+As part of the Data Preview 1 activity {cite:p}`RTN-085` we demonstrated this by creating a DOI for LSSTComCam {cite:p}`10.71929/rubin/2561361`.
+We intend to create DOIs for both LATISS and LSSTCam.
+
+## Data Release DOIs
+
+Rubin Observatory plans to make annual data releases of the Legacy Survey of Space and Time {cite:p}`2019ApJ...873..111I`.
+There will always be an umbrella DOI issued for the entire data release.
+From a citation tracking perspective this is the minimum requirement, but we know that there will be very few papers published that really make use of the full set of imaging and catalog data.
+Furthermore, Independent Data Access Centers (IDACs) {cite:p}`RTN-003` are not going to be taking copies of full data release and instead are likely to restrict their holdings to, for example, the object catalog and deep coadd images, or even a version of the object catalog with a subset of the columns.
+IDACs will be encouraged to create DOIs for their holdings that explicitly link back to the primary data release archive.
+To enable this we intend to issue DOIs to every catalog (object, source, forced source, etc) and every Data Butler {cite:p}`2022SPIE12189E..11J` dataset type (visit images, difference images, co-adds).
+
+An open question is whether to create DOIs for "virtual" Butler data products that can be regenerated on demand.
+
+### Relationships
+
+DOIs much more useful if relationships to other DOIs are specified in the metadata.
+[DataCite](https://support.datacite.org/docs/connecting-to-works) and [Crossref](https://www.crossref.org/documentation/schema-library/markup-guide-metadata-segments/relationships/) specify how relationships between DOIs are specified and we will relate dataset DOIs in the following way:
+
+* The data release dataset "isCollectedBy" the instrument.
+* The catalog and dataset type component DOIs will be declared as "isPartOf" the main data release DOI.
+* If an IDAC creates a DOI for their copy of a component they shall use "isIdenticalTo" the original dataset. (How do we know which was the original version?)
+* If an IDAC has a cut down version of a catalog (e.g., "object lite") then that version "isVariantFormOf" the original full catalog.
+* If a data release includes Butler datasets that contain tabular data (for example in Parquet files) that are also available as catalogs in Qserv, then the catalog DOI "isVariantFormOf" the Butler dataset to indicate that they are almost, but not exactly, the same data.[^dp1_parquet]
+* A new data release "Obsoletes" a previous data release.[^dp1]
+
+
+## References
+
+```{bibliography}
+  :style: lsst_aa
+```
+
+[^dp1]: Data Preview 1 is not obsoleted by Data Preview 2 since they are completely distinct datasets.
+[^dp1_parquet]: Data Preview 1 does not include any tabular butler datasets.
```

Comment:
I don't think this is true.

Reply:
Indeed. I do see that we have "object" parquet files in dp1. In the rsp_lsst_io pages we only have one entry for "Object" as a whole that doesn't distinguish between butler dataset vs searchable catalog.